docker-for-windows/troubleshoot.md

```diff
@@ -332,21 +317,21 @@ Performance tab on the Task Manager:
 
 ![Task Manager](images/virtualization-enabled.png){:width="700px"}
 
-If, at some point, if you manually uninstall Hyper-V or disable virtualization,
-Docker Desktop for Windows cannot start. See: [Unable to run Docker for Windows on
+If you manually uninstall Hyper-V or disable virtualization,
+Docker Desktop cannot start. See [Unable to run Docker for Windows on
 Windows 10 Enterprise](https://github.com/docker/for-win/issues/74).
 
 ### Networking and WiFi problems upon Docker Desktop for Windows install
 
-Some users have encountered networking issues during install and startup of
-Docker Desktop for Windows. For example, upon install or auto-reboot, network adapters
+Some users may experience networking issues during install and startup of
+Docker Desktop. For example, upon install or auto-reboot, network adapters
```

Comment:
"network adapters and/or WiFi may get disabled"

Reply:
Fixed in the review commit.